src/utils.rs

```diff
@@ -95,11 +103,19 @@ impl Color {
             Color::Black => 0,
             Color::Red => 1,
             Color::Green => 2,
-            Color::Yellow => 3,
-            Color::Blue => 4,
+            Color::Yellow => 11,
+            Color::Blue => 12,
             Color::Magenta => 5,
             Color::Cyan => 6,
-            Color::White => 7,
```

Comment:
I think the correct behavior here would be to leave `White` unchanged and to name the newly introduced white `BrightWhite`.